atuin-client/config.toml

```diff
@@ -48,57 +46,68 @@
 ## in any directory within a git repository tree (default: false)
 # workspaces = false
 
-## which filter mode to use when atuin is invoked from a shell up-key binding
-## the accepted values are identical to those of "filter_mode"
-## leave unspecified to use same mode set in "filter_mode"
-# filter_mode_shell_up_key_binding = "global"
+[sync]
```

Comment:
may also want 

```
enabled = true/false
```

some users seem to want confirmation that Atuin is offline.